frontend/src/components/filter-table/filter-table.css

```diff
@@ -74,58 +84,76 @@
     transform: translateX(6px);
     z-index: 1;
 }
+
 .table .resizer.isResizing {
     background: rgba(167, 182, 194, 0.1);
 }
 
 .thead,
 .hidden-row-warning {
-    border-bottom: 1px solid rgba(167, 182, 194, 0.4);
-    box-shadow: 0px 3px 8px -3px rgba(167, 182, 194, 0.5);
     user-select: none;
+    overflow-x: scroll;
 }
+
+::-webkit-scrollbar {
```

Comment:
This concerns me...we shouldn't need prefixed css properties...

Reply:
this is deprecated from when I was trying to implement with native scrollbars. will be removed